lib/tapioca/static/symbol_loader.rb

```diff
@@ -65,7 +65,7 @@ def symbols_from_paths(paths)
 
         private
 
-        sig { returns(T::Array[T.class_of(Rails::Engine)]) }
+        T::Sig::WithoutRuntime.sig { returns(T::Array[T.class_of(Rails::Engine)]) }
```

Comment:
NOTE: the `T::Sig::WithoutRuntime` is needed because tapioca itself does not directly depend on Rails, but will attempt to generate signatures for this method for projects that do not include Rails, and trigger an sig block exception to be logged.